examples/hybrid_ecs/outputs.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  proxy     = "http://${aws_lb.external.dns_name}:8000"
+  admin_api = "http://${aws_lb.external.dns_name}:8001"
```

Comment:
I think from our discussions earlier, this will actually be http to the LB then https from LB to Kong (for the example only) to avoid needing to manage DNS purely for the example

Reply:
Yes you're right, the work I am doing still uses http and port 8000/8001